governance.md

```diff
@@ -0,0 +1,101 @@
+
+The Marketing Lead will be elected according to the same criteria and process as the Director. The Marketing Lead will have the following responsibilities.
+
+- Develop the Marketing portion, including metrics, of the annual operating plan in consultation with the Technical Community. Report regularly to the Steering Committee on progress against plan.
+- Work with the Technical Community and other contributors to develop and execute launch plans for major releases and any major events.
+- Work with the Community Manager and other contributors as needed to support ongoing marketing activities.
+
+The Technical Community Leader and Curriculum Lead’s roles and selection process are described in the Technical Community section above.  
+
+The Committee may choose to define and formalize additional roles to assist the Director or lead particular initiatives under the Voting procedures described below. 
+
+#### Voting Procedures
+
+- 1 vote per Steering Committee Voter
+- Other voting members may be admitted by majority vote of existing Voters.
+- If a Steering Committee Voter misses 3 meetings in a row without designating a proxy, their voting capacity will be suspended until they have attended 2 meetings in succession.
+- If a Steering Committee Voter misses 4 or more meetings in a row, with or without a proxy, the Steering Committee may remove that individual and replace them with someone with greater availability.
+
+Elections and any other voting decisions require a quorum of 50% of voting members. Voting can be held during Committee meetings, or by electronic means within a specified time period.
+
+Amendments to this or Policy documents require a majority vote of the Steering Committee.
+
+### Compliance with Policies and Licenses
+
+Contributors will comply with the policies adopted by the Steering Committee of NRE Labs. These policies may include:
+
+- Contributor License Agreement
+- Code of Conduct
+- Privacy Policy
+- Terms of Use
+- Trademark and Branding Policy
+
+#### Community Assets and Intellectual Property 
+
+Title to all trade or service marks used by the Project (“Project Trademarks”) is currently held by Juniper Networks. In the likely event that the Project joins an open source foundation or fiscal conservancy, Project Trademarks shall be transferred and assigned to that entity to hold on behalf of the Project. Any use of any Project Trademarks by participants in the Project or others shall be in accordance with the Trademark and Branding Policy.
+
+The Project shall develop and own all Project repositories and social media accounts, and domain name registrations created by the Project. The Platform is licensed under the Apache v2.0 license. Lessons contributed to the Project are published under CC BY 2.0. Project documentation, web and marketing content shall be published under Creative Commons license. 
```

Comment:
I'd prefer to see some clear Copyright + License heading rather than a sentence inside that general section: these are probably the most important legal elements that contributors or users (reusers) should be concerned about IMHO (like, "can I copy/paste it to reuse it in class ?", or "can I fork the Antidote/Syringe Go code to create another SaaS startup ?"...)

Reply:
Will moved the license info further up in the doc. There may be a good call for an FAQ describing some of these things in more detail, but not needed in a governance doc as such.